core/flyway/src/main/java/io/thorntail/flyway/impl/FlywayConfiguration.java

```diff
@@ -39,22 +42,22 @@ public void init() {
 		try {
 			dataSourceId = config.getValue(DATASOURCE_PROPERTY, String.class);
 		} catch (Exception e) {
-			FlywayMessages.MESSAGES.dataSourceIdNotConfigured();
+			flywayMessages.dataSourceIdNotConfigured();
```

Comment:
This change does not follow conventions used in the project.

Reply:
This change was only made for unit-tests to function and in someway to challenge the aforementioned conventions =)